tests/gocase/go.mod

```diff
@@ -4,12 +4,14 @@ go 1.23
 
 require (
 	github.com/redis/go-redis/v9 v9.7.0
-	github.com/shirou/gopsutil/v3 v3.24.5
+	github.com/shirou/gopsutil/v4 v4.24.12
 	github.com/stretchr/testify v1.10.0
 	golang.org/x/exp v0.0.0-20241108190413-2d47ceb2692f
-	modernc.org/mathutil v1.6.0
+	modernc.org/mathutil v1.7.1
 )
 
+require github.com/ebitengine/purego v0.8.1 // indirect
```

Comment:
I think we can move this into the `require` clause below.

Reply:
thanks, done